.github/workflows/pull_request.yml

```diff
@@ -1,30 +1,21 @@
-# IMPORTANT NOTE/WARNING!
-# Do not make changes to this file, your changes will be overwritten.
-#
-# This file is automagically generated from:
-# - .github/templates/pull_request.yml.erb
-# - Templates contained in the smartlyio/github-actions-templates repository
-#
-# This file can be updated by editing the template file, and running `devbox render workflows`
-
-name: "Pull Request"
+name: Pull Request
 
 on:
   pull_request:
     branches: [main]
 
 jobs:
   test_action:
```

Comment:
@anttiharju maybe extract `test_action` and `test_unit` into `build.yml` and re-use it?

Reply:
Done in 31daebb93000fe6a9500a3417d89cce2dbe9b80d